tests/kani/Intrinsics/SIMD/Operators/division_float.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! Checks that the `simd_div` intrinsic returns the expected results for floating point numbers.
+//! Checks that the `simd_rem` intrinsic exists.
+#![feature(repr_simd, platform_intrinsics)]
+
+#[repr(simd)]
+#[allow(non_camel_case_types)]
+#[derive(Clone, Copy, PartialEq)]
+pub struct f32x2(f32, f32);
+
+extern "platform-intrinsic" {
+    fn simd_div<T>(x: T, y: T) -> T;
+    fn simd_rem<T>(x: T, y: T) -> T;
+}
+
+#[kani::proof]
+fn test_simd_div() {
+    let dividend: f32 = kani::any::<i8>().into();
```

Comment:
I did this to avoid overflows in the division/restrict the scope of the test. If the division overflows, I would need to adjust the equality test to account for overflow results, however then the harness won't terminate in a reasonable amount of time. Definitely open to suggestions though!

Reply:
What if you restrict the value to a small interval, would that work? Something like:
```rust
let dividend = kani::any_where(|d: f32| d > LOWER_BOUND && d < UPPER_BOUND);
```
I just want to make sure these operations work with non-absolute values.

Reply:
Even with a narrow range of bounds, it doesn't seems to terminate in any reasonable amount of time (at all?). I came up with another solution similar to my first but with multiplying 0.5 by an i8 so that it isn't just round numbers. I'll push my changes a bit later so you can take a look!

Reply:
Awesome. Thanks